src/gkeClient.ts

```diff
@@ -189,11 +191,11 @@ export class ClusterClient {
             cluster: cluster.data.name,
             user: cluster.data.name,
           },
-          name: cluster.data.name,
+          name: contextName,
         },
       ],
       'kind': 'Config',
-      'current-context': cluster.data.name,
+      'current-context': contextName,
```

Comment:
I'm not super familiar with the kubeconfig file syntax, so I'm not sure if any of these other values need to be changed to `contextName`.

Reply:
I think this is it, we should add a test to validate custom name though maybe using `kubectl config current-context`